encrypted-media-respec.html

```diff
@@ -694,15 +694,23 @@
                   </dd>
                   <dt><a def-id="requirement-optional"></a></dt>
                   <dd>
-                    <p>Continue.<!-- Will be updated below. --></p>
+                    <p>If <a def-id="distinctive-identifiers"></a> are not allowed according to <var>restrictions</var>:</p>
+                    <ol>
+                      <li>
+                        <p>Set the <a def-id="option-distinctiveIdentifier"></a> member of <var>accumulated configuration</var> to <a def-id="requirement-not-allowed"></a></p>
+                      </li>
+                      <li>
+                        <p>If the <var>implementation</var> requires a <a def-id="distinctive-identifier"></a> in combination with <var>accumulated configuration</var> and <var>restrictions</var>, return <code>NotSupported</code>.</p>
```

Comment:
This is really executing the "not-allowed" case. I wonder if it might be clearer and less susceptible to rot if we said, "Repeat this step" or "Execute the steps for the `<a def-id="requirement-not-allowed"></a></dt>` case." Thoughts?